content/02.body.md

```diff
@@ -0,0 +1,31 @@
+## Introduction
+
+### Node degree bias
+
+### Unsupervised link prediction
+
+### Feature-degree correlation
+
+
+## Methods
+
+### Network permutation
+
+### XSwap algorithm
+
+### Edge prior
```

Comment:
Should we have separate subsections for xswap edge prior and analytic edge prior?

Reply:
That's how I had it originally. I ended up merging them because most of the analytic went into a supplement (derivation and comparison). But I agree, I think it is best to separate them.